packages/suite/src/reducers/wallet/coinjoinReducer.ts

```diff
@@ -498,7 +498,17 @@ export const selectCurrentTargetAnonymity = memoize((state: CoinjoinRootState) =
     return targetAnonymity;
 });
 
-export const selectIsCoinjoinBlockedByTor = memoize((state: CoinjoinRootState) => {
+export const selectIsCoinjoinGloballyBlockedByTor = memoize((state: CoinjoinRootState) => {
+    const { isTorEnabled } = selectTorState(state);
+
+    if (state.wallet.coinjoin.debug?.coinjoinAllowNoTor) {
+        return false;
+    }
+
+    return !isTorEnabled;
+});
+
+export const selectIsCoinjoinSelectedAccountBlockedByTor = memoize((state: CoinjoinRootState) => {
```

Comment:
Good. FYI, I am taking this a step further in https://github.com/trezor/trezor-suite/pull/7587/commits/e3da67785aac6dcdfb4f632cf3220a1d1780b691 and remove the selector for selected account.